gatsby-browser.js

```diff
@@ -1 +1,3 @@
 export { wrapRootElement } from "./wrapRootElement"
+import wrapWithProvider from "./wrap-with-provider"
+export const wrapRootElement = wrapWithProvider
```

Comment:
action: you are replacing `wrapRootElement` with `wrapWithProvider`, which we will no logger wrapping with the providers inside`wrapRootElement`
a proper way to do that is putting them all together into a single file